ELECTIONS.md

```diff
@@ -0,0 +1,201 @@
+# OpenSearch Technical Steering Committee Election Process
+
+## Overview
+
+This document establishes the election procedures for the OpenSearch Technical Steering Committee (TSC) as referenced in Section 3.e of the [OpenSearch Project Technical Charter](https://opensearch.org/wp-content/uploads/2025/02/OpenSearch-Project-Technical-Charter-Final-9-13-2024.docx.pdf). The TSC will hold elections annually to fill expiring terms and maintain continuity of technical oversight for the OpenSearch Project. The committee size includes 15 members.
+
+## Election Timeline
+
+### Annual Election Schedule
+
+- **End of term**: Facilitator opens a GitHub issue for community nominations for 1 week
+- **Week 1**: Call for nominations opens; exception request period opens
+- **Prior to next TSC Meeting**: Nominations and exception requests close; candidate statements collected by facilitator
+- **During TSC Meeting**: Voting by current TSC members will take place at next scheduled TSC meeting
+- **Within 3 Days of Vote**: Results announced and certified
+- **Implementation**: New TSC members take office at the next scheduled TSC meeting
+- **Chair Election**: During the first meeting with new TSC members, a chair will be nominated and voted upon. 
+
+Elections will be held annually, with the timing determined by the TSC to ensure adequate transition time before expiring terms end.
+
+## Facilitator
+
+### Responsibilities
+Election Facilitator is responsible for:
+- Publishing all election announcements and communications
+- Maintaining and publishing the eligible voter list
+- Processing voter eligibility exception requests
+- Preparing and distributing electronic ballots
+- Assisting candidates with statement preparation and publication
+- Monitoring the voting process for integrity
+- Tallying results according to the procedures in this document
+- Announcing results and certifying the election
+
+## Voter Eligibility
+
+### Primary Eligibility Criteria
+The first voting of new member elections will be held by the TSC. This is to get through the process successfully for its first voting iteration, and to ensure goals of electing new members is completed in a timely manner. This will be amended in the future.
```

Comment:
Amended in the future seems open ended, why include this or you should put timing in here. Do you think we should involve the community in voting?

Reply:
Definitely something to consider @KarstenSchnitter @jkowall - another thing to talk through at the TSC meeting would be - voting done by **Maintainers** (listed in published MAINTAINERS.md file as of date the nominations open)